FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -333,5 +350,126 @@ public void SelectInfinity(string query, double doubleExpected, float floatExpec
             Assert.That(reader.Read(), Is.EqualTo(false));
             conn.Close();
         }
+
+        [Test]
+        public void CreateDropFillTable()
+        {
+            using (var conn = new FireboltConnection(USER_CONNECTION_STRING))
+            {
+                conn.Open();
+                CreateCommand(conn, CREATE_TABLE).ExecuteNonQuery();
+                DbCommand insert = CreateCommand(conn, INSERT);
+                insert.Prepare();
+                insert.Parameters.Add(CreateParameter(insert, "@i", 1));
+                insert.Parameters.Add(CreateParameter(insert, "@n", 2));
+                insert.Parameters.Add(CreateParameter(insert, "@bi", 1234567890L));
+                insert.Parameters.Add(CreateParameter(insert, "@r", 3.14));
+                insert.Parameters.Add(CreateParameter(insert, "@dp", 2.718281828));
+                insert.Parameters.Add(CreateParameter(insert, "@t", "hello"));
+                insert.Parameters.Add(CreateParameter(insert, "@d", "2023-10-18"));
+                insert.Parameters.Add(CreateParameter(insert, "@ts", "2023-10-18 17:30:20.123456"));
+                insert.Parameters.Add(CreateParameter(insert, "@tstz", "2023-10-18 17:30:20.123456+02"));
+                insert.Parameters.Add(CreateParameter(insert, "@b", true));
+                insert.ExecuteNonQuery();
+
+                DbCommand selectAll = CreateCommand(conn, SELECT_ALL);
+
+                using (DbDataReader reader = selectAll.ExecuteReader())
+                {
+                    Assert.That(reader.GetFieldType(0), Is.EqualTo(typeof(int)));
```

Comment:
Just a suggestion, can we transform this piece to a for loop?

Reply:
In similar places as well

Reply:
done for the metadata